apps/root/src/pages/transfer/components/transfer-button/index.tsx

```diff
@@ -56,10 +62,25 @@ const TransferButton = ({ disableTransfer, onTransferClick }: TransferButtonProp
     </Button>
   );
 
+  const ReconnectButton = (
+    <Button fullWidth variant="contained" onClick={() => openConnectModal()}>
+      <FormattedMessage description="reconnect wallet" defaultMessage="Reconnect wallet" />
+      {authWallet && (
+        <>
+          {' ('}
+          <Address address={authWallet.address} trimAddress />
```

Comment:
Same as before, plus we should be able to send this into the formatted message as a value